TESTS/events/queue/main.cpp

```diff
@@ -7,6 +7,7 @@
 
 using namespace utest::v1;
 
+#define TEST_EQUEUE_SIZE 1024
```

Comment:
@bridadan pointed this out to me. Looks great and matches the way we're handling stacks here 👍